packages/DatePicker/src/DatePicker.js

```diff
@@ -26,9 +26,6 @@ const propTypes = {
   /** Date format used while displaying date. It should be human-friendly and spelled out, default is MMMM DD,YYYY */
   humanFormat: PropTypes.string,
 
-  /** ID for the `<input>`. */
-  id: PropTypes.string,
```

Comment:
Unrelated change here: I suggested we remove the `id` from the datePicker component (which is spread onto the `Input` subcomponent, so that we encourage consumers to use the `<DatePicker.Input id={id}>` instead and can add the id there.

Apart from being easier to understand and be more explicit, this avoids the situation where an id maybe applied to root component, but is overridden by `<DatePicker.Input {...datePickerA11yProps}>`

Reply:
`datePickerA11yProps` → `a11yProps` (from the FormElement.Content render prop argument).  Still a valid argument, but maybe @allison-c or @DanilAgafonov would have some concerns with this change to the API since they are more familiar with the `<DatePicker>`?

Reply:
👍 I'm ok with it